akka-projection-core/src/main/scala/akka/projection/ProjectionSettings.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.duration._
+
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.InternalApi
+import akka.util.JavaDurationConverters._
+import com.typesafe.config.Config
+
+trait ProjectionSettings {
+
+  def minBackoff: FiniteDuration
+  def maxBackoff: FiniteDuration
+  def randomBackoffFactor: Double
```

Comment:
maybe worth grouping these 3 together in a `case class RestartSettings`?

Reply:
can we add this when (if) we have other settings?

Reply:
I think it's value to group them because they should typically be set together. Otherwise
```
def withBackoff(min, max, randomFactor)
```

Reply:
I introduced `withBackoff(min, max, randomFactor)`. Glad to add a case class if/when the settings grow.